src/lib.rs

```diff
@@ -491,8 +569,14 @@ fn bad_file() {
 #[test]
 fn truncated() {
     let path = Path::new("test/Truncated.mp3");
-    let duration = from_path(path).unwrap();
-    assert_eq!(206, duration.as_secs());
-    let nanos = duration.subsec_nanos();
-    assert!(7 * 100_000_000 < nanos && nanos < 8 * 100_000_000);
+    let error = from_path(path).unwrap_err();
+
+    if let ErrorKind::UnexpectedEOF = error.kind {
```

Comment:
Nitpick: Can you use assert_eq!() instead of a if-branch + panic!?

Reply:
I tried that, but it seems that ErrorKind has to be Eq or PartialEq and simply deriving it didn't work(io:Error is not PartialEq) so I thought this is simpler.

Or does assert_eq! also work on regular enums?